FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp

```diff
@@ -1350,4 +1350,254 @@ void OpDispatchBuilder::AVX128_VPSLLDQ(OpcodeArgs) {
   AVX128_ShiftDoubleImm(Op, ShiftDirection::LEFT);
 }
 
+void OpDispatchBuilder::AVX128_VINSERT(OpcodeArgs) {
+  const auto DstSize = GetDstSize(Op);
+  const auto Is128Bit = DstSize == Core::CPUState::XMM_SSE_REG_SIZE;
+  const auto Selector = Op->Src[2].Literal();
```

Comment:
This had a bitmask dropped which needs to be ketpt.

Reply:
4 eyes are better than 2.. thanks!